.gitignore

```diff
@@ -22,4 +22,9 @@ config/deploy/deployment.yaml
 config/deploy/webhook/deployment.yaml
 config/deploy/webhook/service.yaml
 config/olm/addon-operator.csv.yaml
+config/deploy/ocm-api-mock/deployment.yaml
```

Comment:
this line is not needed. the folder was renamed to `api-mock` and should already be gitignored AFAIK

Reply:
removed the line